packages/block-editor/src/components/block-toolbar/index.js

```diff
@@ -85,6 +86,12 @@ export default function BlockToolbar( { hideDragHandle } ) {
 		transform: shouldShowMovers ? 'translateX(0px)' : 'translateX(100%)',
 	};
 
+	const animatedBlockParentStyles = {
+		opacity: shouldShowMovers ? 1 : 0,
+		transform: shouldShowMovers ? 'translateY(-63px)' : 'translateY(0)',
+		transitionDelay: shouldShowMovers ? 0 : 0,
```

Comment:
It can be simplified to:
```suggestion
		transitionDelay: 0,
```